test/bin/manage_brew_rpms.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# Note: Avoid sourcing common.sh or common_version.sh in this script to allow
+# its execution in a containerized environment with limited set of tools.
+
+usage() {
+    echo "Usage: $(basename "$0") [access | download <version> <path>]"
+    echo "  download:   Download the RPM version to the path as specified"
+    echo "  access:     Exit with non-zero status if brew cannot be accessed"
+}
+
+action_access() {
+    if ! curl -I "https://brewhub.engineering.redhat.com" &>/dev/null ; then
+        return 1
+    fi
+    return 0
+}
+
+action_download() {
+    local -r ver=$1
+    local -r dir=$2
+
+    if ! action_access ; then
+        echo "ERROR: Brew Hub site is not accessible"
+        exit 1
+    fi
+
+    # Attempt downloading the specified build version
+    local package
+    package=$(brew -q list-builds --package=microshift --state=COMPLETE | grep "^microshift-${ver}" | sort | tail -1) || true
+    if [ -z "${package}" ] ; then
+        echo "ERROR: Cannot find MicroShift '${ver}' packages in brew"
+        exit 1
+    fi
+
+    package=$(awk '{print $1}' <<< "${package}")
+    echo "Downloading '${package}' packages from brew"
+
+    mkdir -p "${dir}"
+    pushd "${dir}" &>/dev/null
+    brew download-build --arch="$(uname -m)" "${package}"
+    popd &>/dev/null
+}
+
+#
+# Main
+#
+if [ $# -lt 1 ] ; then
+    usage
+    exit 1
+fi
+action="${1}"
+
+"${SCRIPTDIR}/../../scripts/fetch_tools.sh" brew
```

Comment:
Nit: this is required only for download action, not access, but I don't think it hurts here